CRT.c

```diff
@@ -915,9 +915,8 @@ void CRT_setMouse(bool enabled) {
 #endif
 
 void CRT_init(const Settings* settings, bool allowUnicode) {
-   redirectStderr();
-
    initscr();
+   redirectStderr();
```

Comment:
It's not clear to me what problem this change is solving.  Maybe a note in the commit message for this aspect?

Reply:
when htop is run e.g. via `env -i htop` it just exists without any message, cause `initscr()` does not return and we have redirected stderr and the signal handlers to dump stderr are not yet installed (since they also cleanup up ncurses).